lighthouse-core/test/audits/font-display-test.js

```diff
@@ -231,6 +231,8 @@ describe('Performance: Font Display audit', () => {
   });
 
   it('handles varied font-display declarations', async () => {
+    // Make sure we don't use sourceURL when it's not a valid URL, see https://github.com/GoogleChrome/lighthouse/issues/8534
+    stylesheet.header.sourceURL = 'custom-attribute';
```

Comment:
any chance you could split this off into its own test? I could see us revising `font-display` at some point and having to reconfigure the tests and losing this very important line :)

Reply:
sure